### ISSUE TITLE
Language switch: label and accessible name don't match

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#5108]
The language switch contains  an `aria-label` that does not match or include the visible text of the control. This is an issue for speech navigation where users dictate the text of the element they see and the assistant tries to find this element by it's accessible name (the aria-label if set). If these don't match, speech input fails to locate the control. 

More info: https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.

### Tasks
- [ ] Update recommendations on labelling the language switch so it's conform with [SC 2.5.3](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name)
